Admin/Infrastructure/Harvister/Network Creation/README.md

```diff
@@ -64,6 +72,15 @@ Use the `ip` command to list the interfaces on your machine. We will need the *M
     <img src="Images/IPLINK.png">
 
 
+> [!NOTE]
+> The use of the following options is **OPTIONAL** you can remove all of them, and the netplan can be applied to multiple VMs without issues.
+> ```
+> match:
+>     macaddress: MACADDR
```

Comment:
Make a note that "MACADDR" is a placeholder for an actual mac address
"FF:FF:FF:FF:FF:FF"